app/uk/gov/hmrc/incorporatedentityidentificationfrontend/featureswitch/core/config/FeatureSwitchingModule.scala

```diff
@@ -42,3 +42,8 @@ case object BusinessVerificationStub extends FeatureSwitch {
   override val configName: String = "feature-switch.business-verification-stub"
   override val displayName: String = "Use stub for Business Verification flow"
 }
+
+case object CotaxNotFoundStub extends FeatureSwitch {
```

Comment:
This isn't a stub, it's allowing a new submission of an unverified/unmatched ctutr. A better name might be something like EnableUnmatchedCtutrJourney